### PR TITLE
Various updates based on issues

### PR DESCRIPTION
**Pull request by dabit3**
Addresses following issues:

#2529 
#2543 
#2560 
#2517 
#2502 
#1188

### CODE REVIEW

File: docs/lib/auth/fragments/js/manageusers.md
Comment (mauerbac) [inline docs/lib/auth/fragments/js/manageusers.md:105]: nit: what is sub here?

**Review by mauerbac — APPROVED** [review]
lgtm, will wait for another review before merging

File: docs/lib/auth/fragments/js/manageusers.md
Comment (amhinson) [reply docs/lib/auth/fragments/js/manageusers.md:105]: Short for subject (OIDC spec)

**Review by amhinson — APPROVED** [review]

File: docs/start/getting-started/fragments/reactnative/setup.md
Comment (ericclemmons) [inline docs/start/getting-started/fragments/reactnative/setup.md:14]: Thanks for this!  A small tweak, but helps reduce confusion.

**Review by ericclemmons — APPROVED** [review]
Thanks for these fixes!

File: docs/start/getting-started/fragments/angular/auth.md
Comment (undefobj) [inline docs/start/getting-started/fragments/angular/auth.md:48]: Nit: These things are *basically* the same now that Angular CLI uses webpack but they used to be different. For older Angular CLIs this won't be the same result.

**Review by undefobj — APPROVED** [review]
One minor nit but LGTM.

**Review by yuth — APPROVED** [review]
LGTM!

**Pull request merged**
